framework/src/engine/consensus/index.ts

```diff
@@ -13,13 +13,13 @@
  */
 
 export { Consensus } from './consensus';
+export * from './constants';
```

Comment:
we can revert back to the original exports from constants as we don't want to expose p2p rpc endpoints constants